Cargo.toml

```diff
@@ -36,6 +36,10 @@ toml = "0.1"
 url = "0.2"
 winapi = "0.1"
 
+[dependencies.file-lock]
+git = "https://github.com/Byron/file-lock"
+branch = "cargo-adjustments"
```

Comment:
I'll see if I can't set up appveyor CI for cargo, that should give us some Windows coverage for PRs at least.